component/src/main/java/com/synopsys/integration/alert/component/diagnostic/database/DefaultDiagnosticAccessor.java

```diff
@@ -101,7 +106,34 @@ private NotificationDiagnosticModel getNotificationDiagnosticInfo() {
         long numberOfNotifications = notificationContentRepository.count();
         long numberOfNotificationsProcessed = notificationContentRepository.countByProcessed(true);
         long numberOfNotificationsUnprocessed = notificationContentRepository.countByProcessed(false);
-        return new NotificationDiagnosticModel(numberOfNotifications, numberOfNotificationsProcessed, numberOfNotificationsUnprocessed);
+        List<ProviderNotificationCounts> providerNotificationCounts = getProviderNotificationCounts();
+        return new NotificationDiagnosticModel(numberOfNotifications, numberOfNotificationsProcessed, numberOfNotificationsUnprocessed, providerNotificationCounts);
+    }
+
+    private List<ProviderNotificationCounts> getProviderNotificationCounts() {
+        Set<Long> providerConfigIds = new LinkedHashSet<>();
+        int pageSize = 100;
```

Comment:
In searching the code, I see a mix of page sizes being set as 100 or 200. Any specific reason to use 100 here vs 200?

Reply:
the size of 200 was updated for processing notifications.  Most other places it's still 100.  I will create a ticket to update this to be consistent across the board in Alert.